docs/applied_prompting/recipes.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_position: 2
+---
+
+# Creating Recipes
+
+If you have ever searched for a recipe online, you will understand how overwhelming it can be to find one that's relatively simple, uses the ingredients you have in your kitchen, and meets your dietary needs. ChatGPT can help with this. It can become your personal assistant in the kitchen, helping you create delicious recipes tailored to your specific needs. ChatGPT is a gourmet chef!
+
+Let's look at a basic example to get a taste of what ChatGPT is capable of:
+
+<pre>
+    Please create a recipe for a sauce that can be served with steak.
+</pre>
+
+<pre>
+    Sure! Here is a recipe for a simple and delicious steak sauce:
+
+    Ingredients:
+
+    1/2 cup beef broth
+    2 tablespoons Worcestershire sauce
+    2 tablespoons red wine vinegar
+    1 tablespoon Dijon mustard
+    2 teaspoons cornstarch
+    1/2 teaspoon garlic powder
```

Comment:
Fix spacing here, when expanded looks odd. Try using '''text ''', but ` instead of '